### PR TITLE
Getting started

### PR DESCRIPTION
**Pull request by oligriffiths**
First stab at a getting started guide.

`yarn serve` to test

### CODE REVIEW

File: src/docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+A `Brocfile.js` in the project root tells Broccoli how to build your project's assets. It can be as simple or as
+complicated as your project needs. For this example, we're going to keep it relatively simple and just build Sass into
+CSS and Coffeescript into Javascript. We're also going to assume your project structure looks like:
+
+In your `package.json` add the following to the `scripts` node (add this if it's not present):
+
+```json
+{
+  "scripts": {
+    "build": "broccoli build --overwrite",
```
Comment (stefanpenner) [inline src/docs/getting-started.md]: Let's remove `--overwrite` from broccoli, it feels out-of-place.

File: src/docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+To run a build, run `yarn build` (if you added the script) or `broccoli build --overwrite` (note: without `--overwrite`
+the contents of the output directory `dist` will NOT be overwritten and produce an error).
+
+You should see something like:
+
+```sh
+$ yarn build
+$ broccoli build --overwrite
+
```
Comment (stefanpenner) [inline src/docs/getting-started.md]: We should fix broccoli to not emit all this whitespace

File: src/docs/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+$ cat dist/index.html
+Hello World!
+```
+
+Now try running `yarn serve` or `broccoli serve` and you should see:
+
+```sh
+$ broccoli serve
+Serving on http://localhost:4200
+
```
Comment (stefanpenner) [inline src/docs/getting-started.md]: ^

**Comment by stefanpenner** [comment]
This looks like a great start, I suspect though we may need to work with learning team folks, and mike/tom when it comes to the structure of successful tutorials.

I don't think that blocks anything here, as a first iteration totally gives them sometime to work with.
Let me know when you feel comfortable with me reaching out to them, maybe we can steal some of their time/ideas/suggestions.

**Comment by oligriffiths** [comment]
@stefanpenner reach away. The issue with removing overwrite is that users will have a worse experience. I’d say leave it here, fix it in broccoli, then remove from guides

**Comment by stefanpenner** [comment]
cc @mnorth / @tomdale your input, if you have cycles, would be appreciated.

**Comment by stefanpenner** [comment]
Not something to block this effort, but thoughts on using http://codesandbox.io to provide low-friction way for folks to run the demo/tutorial steps? @mnorth introduced me to it, he may have more insight here...

File: src/docs/getting-started.md [hunk 4]
```diff
@@ -134,3 +134,14 @@ You will see the URL `http://localhost:4200`, if you open this in the browser yo
 Browser.
 
 Congratulations 👏, you've just written your first Broccoli pipeline!
+
+For a demo, checkout this sanbox from [codesandbox.io](https://codesandbox.io)
+
+<div class="mobile-show">
+<a href="https://codesandbox.io/s/github/broccolijs/broccoli-tutorial/tree/master/">
```
Comment (stefanpenner) [inline src/docs/getting-started.md]: this makes me pretty motivated to add livereload :P

Comment (oligriffiths) [reply src/docs/getting-started.md]: Yeah, live reload should be part of the core.
Can we use https://github.com/broccolijs/broccoli/issues/352 to setup a broccoli 2.0 roadmap? Or should we make a new issue?

Comment (stefanpenner) [reply src/docs/getting-started.md]: One approach would be to re-visit as part of (or after) unifying the watcher effort.

Comment (stefanpenner) [reply src/docs/getting-started.md]: But if you see a fast-path, and are motivated. Please don't let me block you.

Comment (oligriffiths) [reply src/docs/getting-started.md]: I'm not sure what you're asking me to do tbh, is there an issue with using this now?

Comment (stefanpenner) [reply src/docs/getting-started.md]: sorry, i misread:

> is there an issue with using this now?

No issue

**Comment by stefanpenner** [comment]
🕺

**Pull request merged**